File: cfgs/java_cfgs_foo/MAXIMUM_REMOVAL_FROM_ARRAY_WHEN_REMOVAL_TIME_WAITING_TIME.java
Nodes
1;special;;;
2;statement;int count = 0;14;
3;statement;int cummulative_sum = 0;15;
4;statement;fun_1(foo_1);16;
5;statement;int i = 0;17;
6;conditional;foo_1 < foo_2;18;
7;conditional;foo_1[foo_2] >= foo_3;20;
8;statement;count++;21;
9;statement;cummulative_sum += foo_1[foo_2];22;
10;statement;i++;19;
11;statement;return foo_1;25;
12;exit;;;
Edges
0;2;
1;3;
2;4;
3;5;
4;6;
5;7;11;
6;8;10;
7;9;
8;10;
9;6;
10;12;
11;

File: cfgs/pythoncfgs/MAXIMUM_REMOVAL_FROM_ARRAY_WHEN_REMOVAL_TIME_WAITING_TIME.py
Nodes
1;statement;foo_1 = 0;8;
2;statement;foo_1 = 0;9;
3;statement;fun_1();10;
4;conditional;fun_1(foo_1);11;
5;statement;return foo_1;15;
6;conditional;foo_1[foo_2] >= foo_3;12;
7;statement;foo_1 += 1;13;
8;statement;foo_1 += foo_2[foo_3];14;
9;exit;;
Edges
0;2;
1;3;
2;4;
3;5;6;
4;9;
5;4;7;
6;8;
7;4;
8;